vms/platformvm/txs/executor/state_changes.go

```diff
@@ -300,7 +300,7 @@ func advanceDynamicFeeState(
 	changes.SetFeeState(dynamicFeeState)
 }
 
-// advanceValidatorFeeState advances the validator fee state by [seconds]. SoVs
+// advanceValidatorFeeState advances the validator fee state by [seconds]. L1 validators
 // are read from [parentState] and written to [changes] to avoid modifying state
 // while an iterator is held.
```

Comment:
```suggestion
// advanceValidatorFeeState advances the validator fee state by [seconds]. L1
// validators are read from [parentState] and written to [changes] to avoid
// modifying state while an iterator is held.
```